.github/ISSUE_TEMPLATE/gcsfuse-pull-request.md

```diff
@@ -0,0 +1,7 @@
+This will be pull request template
+### Description
```

Comment:
add one more field - Link to the issue incase of bug fix.

Reply:
fixed